tests/integration/test_active_response/test_execd/test_execd_firewall_drop.py

```diff
@@ -0,0 +1,204 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import hashlib
+import os
+import platform
+import pytest
+import time
+import requests
+import subprocess
+import yaml
+import json
+import socket
+
+from wazuh_testing.tools import WAZUH_PATH, LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.authd_sim import AuthdSimulator
+from wazuh_testing.tools.remoted_sim import RemotedSimulator
+from conftest import *
+from subprocess import Popen, PIPE, STDOUT
+
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=0), pytest.mark.agent]
+
+CLIENT_KEYS_PATH = os.path.join(WAZUH_PATH, 'etc', 'client.keys')
+SERVER_KEY_PATH = os.path.join(WAZUH_PATH, 'etc', 'manager.key')
+SERVER_CERT_PATH = os.path.join(WAZUH_PATH, 'etc', 'manager.cert')
+CRYPTO = "aes"
+SERVER_ADDRESS = 'localhost'
+PROTOCOL = "tcp"
+
+test_metadata = [
+    {
+        'command': 'firewall-drop5',
+        'rule_id': '5715',
+        'ip': '3.3.3.3',
+        'results': {
+            'success': True,
+        }
+    },
+    {
+        'command': 'firewall-drop0',
+        'ip': '3.3.3.3',
+        'rule_id': '5715',
+        'results': {
+            'success': False,
+        }
+    },
+]
+
+params = [
+    {
+        'CRYPTO': CRYPTO,
+        'SERVER_ADDRESS': SERVER_ADDRESS,
+        'REMOTED_PORT': 1514,
+        'PROTOCOL': PROTOCOL
+    } for _ in range(0, len(test_metadata))
```

Comment:
```suggestion
    } for _ in range(len(test_metadata))
```
`range` always starts from 0